nette/notejam/app/presenters/templates/Note/delete.latte

```diff
@@ -0,0 +1,9 @@
+{block title}{$note->name}{/block}
+{block header}{$note->name}{/block}
+{block content}
+	<p>Are you sure you want to delete the note?</p>
+	{form deleteNoteForm}
```

Comment:
if you wanna add classes, it's nicer to have it as 

```html
<form n:name="deleteNoteForm" class="custom-class">
```

which for example the phpstorm autocompletes, because it's a html attribute, not custom macro attribute. It also allows me to invoke "go to definition" into the css file.

While this macro allows none of that

```html
{form deleteNoteForm, class => "custom-class"}
```

Because of that, the `n:` macros are superior to "smarty macros" syntax, even through the're equivalent.